data/recipes/workspace_user_login_ts.json

```diff
@@ -39,6 +42,9 @@
     ["--filter_expression", "Filter expression to use to query Workspace logs. See https://developers.google.com/admin-sdk/reports/reference/rest/v1/activities/list",""],
     ["--incident_id", "Incident ID (used for Timesketch description).", null],
     ["--sketch_id", "Timesketch sketch to which the timeline should be added.", null, {"format": "regex", "regex": "^\\d+$"}],
+    ["--timesketch_endpoint", "Timesketch endpoint", null],
```

Comment:
Should we set this to the default Timesketch URL? ```http://127.0.0.1:5000``` or will this be read from the ```data/config.json``` file?

Reply:
It's very likely to be replaced by *some* config file, but we can set a default.